src/pint/models/solar_wind_dispersion.py

```diff
@@ -23,42 +23,41 @@ class SolarWindDispersion(Dispersion):
     ----------
     Edwards et al. 2006, MNRAS, 372, 1549; Setion 2.5.4
     Madison et al. 2019, ApJ, 872, 150; Section 3.1.
-
     """
 
     register = True
     category = "solar_wind"
 
     def __init__(self):
-        super(SolarWindDispersion, self).__init__()
+        super().__init__()
         self.add_param(
             floatParameter(
                 name="NE_SW",
                 units="cm^-3",
                 value=0.0,
                 aliases=["NE1AU", "SOLARN0"],
-                description="Solar Wind Parameter",
+                description="Solar Wind density at 1 AU",
             )
         )
         self.add_param(
             floatParameter(
-                name="SWM", value=0.0, units="", description="Solar Wind Model"
+                name="SWM",
+                value=0.0,
+                units="",
+                description="Solar Wind Model (0 is the only option)",
```

Comment:
How about "Solar Wind Model Selection (0 = Edwards et al. 2006, more to come)"

Reply:
Is `SWM` a new type of solar wind model other than the spherical one? If that is the case, I would say make it a new component. Of if it is an alias, we should just add it to the parameter alias.